src/content/members/core/danieltott.ts

```diff
@@ -9,6 +9,8 @@ export const danieltott: MemberObject = {
 	//
 	// Name - If not defined here, it will default to your display name on GitHub. If that's not defined, then your GitHub username.
 	name: 'Dan Ott',
+	// emoji
+	emoji: '🎃',
```

Comment:
I really preferred the canoe

```suggestion
	emoji: '🎃',
```

Reply:
We're gonna stay with pumpkin